### PR TITLE
Updated hashes of regional_workflow and UFS_UTILS

### PR DESCRIPTION
**Pull request by mdtoy**
## DESCRIPTION OF CHANGES: 
This updates the hashes of regional_workflow and UFS_UTILS to work with [PR #448](https://github.com/NOAA-EMC/regional_workflow/pull/448) into regional_workflow

## TESTS CONDUCTED: 
See [PR #448](https://github.com/NOAA-EMC/regional_workflow/pull/448) into regional_workflow

## DOCUMENTATION:
If this PR is contributing new capabilities that need to be documented, please also include updates to the RST files (docs/UsersGuide/source) as supporting material.

### CODE REVIEW

File: Externals.cfg [hunk 1]
```diff
@@ -11,8 +11,7 @@ required = True
 protocol = git
 repo_url = https://github.com/NOAA-EMC/UFS_UTILS
 # Specify either a branch name or a hash but not both.
-#branch = develop
-hash = 005f9a0a
+hash = ea821358
 local_path = src/UFS_UTILS
```
Comment (JeffBeck-NOAA) [inline Externals.cfg:15]: Hi @mdtoy, can you keep the "#branch = develop" line in your version?

**Comment by JeffBeck-NOAA** [comment]
@mdtoy, now that your regional_workflow PR is merged, you can update the hash of that repo in the Externals.cfg file, and I think we're good to go after that.

**Comment by JulieSchramm** [comment]
The hash for UFS_UTILS has already been updated to ea821358 in the master branch.  The regional_workflow hash needs to be updated.

**Comment by JeffBeck-NOAA** [comment]
> The hash for UFS_UTILS has already been updated to ea821358 in the master branch. The regional_workflow hash needs to be updated.

Thanks, @JulieSchramm!  @mdtoy, it looks like you can remove the UFS_UTILS hash from this PR (it went in with this [PR](https://github.com/ufs-community/ufs-srweather-app/pull/135), and just update the regional_workflow hash.

**Comment by JeffBeck-NOAA** [comment]
The hash changes mentioned here were covered in previous PRs, so closing this PR.

**Pull request closed without merging**